src/org/rascalmpl/test/infrastructure/RascalJUnitTestRunner.java

```diff
@@ -283,7 +280,7 @@ private Description getDescription(String name, ISourceLocation loc) {
 
         @Override
         public void start(String context, int count) {
-            notifier.fireTestRunStarted(module);
+            // notifier.fireTestRunStarted(module);
```

Comment:
are we sure we want to disable these notifications?

Reply:
Yes, these are actually not allowed to be called anymore. They've been kept in for binary backwards compatibility but the javaDoc says "do not invoke" literally.